src/elements/content-explorer/stories/tests/DeleteConfirmationDialog-visual.stories.js

```diff
@@ -4,44 +4,24 @@ import { addRootElement, defaultVisualConfig } from '../../../../utils/storybook
 
 import DeleteConfirmationDialog from '../../DeleteConfirmationDialog';
 
-// need to import this into the story because it's usually in ContentExplorer
-import '../../../common/modal.scss';
-
 const item = {
     id: '123456',
-    name:
-        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Eget nulla facilisi etiam dignissim diam quis enim lobortis scelerisque. Aliquam faucibus purus in massa tempor nec. Ut consequat semper viverra nam libero justo laoreet sit amet. Purus gravida quis blandit turpis cursus in hac. Dui ut ornare lectus sit amet est. Nisl condimentum id venenatis a condimentum vitae sapien ',
+    name: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Eget nulla facilisi etiam dignissim diam quis enim lobortis scelerisque. Aliquam faucibus purus in massa tempor nec. Ut consequat semper viverra nam libero justo laoreet sit amet. Purus gravida quis blandit turpis cursus in hac. Dui ut ornare lectus sit amet est. Nisl condimentum id venenatis a condimentum vitae sapien ',
 };
 
 export const deleteDialogNotLoading = {
     render: () => {
-        const { appElement, rootElement } = addRootElement();
-
-        return (
-            <DeleteConfirmationDialog
-                appElement={appElement}
```

Comment:
we don't need to pass `appElement` anymore? it looks required in the component

Reply:
for the stories, we dont need it to open the modal